builder_client_helpers/examples.go

```diff
@@ -30,16 +30,16 @@ func ListExamplesPath() string {
 }
 
 // ListExamples provides a list of all the builtin examples
-func (c *Client) ListExamples(ctx context.Context, path string, maintainer *string, type_ *string) (*http.Response, error) {
```

Comment:
Use `libType` instead of `type1`

Reply:
Done 3b2517935dda8b02d5eb42c7e42100b0841c89db